### DISCUSSION TITLE
Is it Possible to Get Doorbell TVOS Alerts From One Floodlight But Not Another?

### DISCUSSION THREAD

**Discussion by CaseyTal** [dgreif/ring#714] [Q&A] [answered]
Hello!

I have a Ring Alarm system and two Ring Floodlight cameras. Everything works great!

One Camera is for the backyard and the other covers the driveway. I would like to get motion alerts/doorbell rings when a car pulls into the driveway, but I don't want motion/doorbell alerts from the dogs going to the backyard. 

Is there a way to only get motion/doorbell alerts for one camera and not the other? I tried fiddling with the Device Types and IDs and couldn't fully understand what to do or if it's possible.

**Comment by dgreif** [comment, marked as answer]
This is not currently possible. I don't plan to add it because the rumor is that TvOS 15 will add native support for this behavior on motion.
